src/components/formSections/components/TermsListField.js

```diff
@@ -259,28 +328,46 @@ export default class TermsListField extends React.Component {
     let termNoteError = false;
 
     const termsList = terms.map((term, i) => {
-      const errorMessage = this.validateNoteField(value, term.value);
+      const errorMessage = this.validateFields(value, term.value);
       termNoteError = errorMessage ? true : termNoteError;
 
       return (
-        <React.Fragment key={term.value}>
+        <Card
+          data-test-term
+          headerStart={
+            <strong data-test-header={term.name}>
+              <FormattedMessage id="ui-licenses.term" />
```

Comment:
We shouldn't construct strings like this by just appending a number to a string because it'll render incorrectly for RTL languages. Instead, use a separate translation string like the way `stripes-erm-components.doc.title` is used.